MapboxGeocoder.xcodeproj/project.pbxproj

```diff
@@ -1135,7 +1157,9 @@
 				PRODUCT_NAME = MapboxGeocoder;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_SWIFT3_OBJC_INFERENCE = Off;
+				SWIFT_VERSION = 5.0;
```

Comment:
I have no idea how this happened. Maybe a dirty checkout of the project that I forgot about. I’ll revert these unnecessary changes.